src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -1,184 +1,347 @@
+</figure>
+
+The last step in the performance improvement workflow cycle is two-fold:
 
-We will cover more about our Core Web Vitals tooling at [web.dev Live](/live/) in June. Sign up to get updates on the event!
+1. Use [CrUX on BigQuery](/chrome-ux-report-bigquery/), [CrUX API](/chrome-ux-report-api/), [PageSpeed Insights API](https://developers.google.com/speed/docs/insights/v5/get-started), and/or the [`web-vitals`](https://www.npmjs.com/package/web-vitals) JavaScript library to automate a website's field data collection, and optionally use this data to power custom dashboards and alerting systems.
+2. Use [Lighthouse-CI](https://github.com/GoogleChrome/lighthouse-ci) for performance regression testing.
+
+{% Aside %}
+[Research by Google](https://youtu.be/YJGCZCaIZkQ?t=112) has shown that most performance improvements tend to regress within six months. A website needs continuous monitoring in both the lab _and_ the field to identify worsening trends in Core Web Vitals and other performance metrics to avoid regressions.
+{% endAside %}
 
-~ by Elizabeth and Addy, WebPerf Janitors
+The tools listed in the first step may seem like a lot, but you don't necessarily need _all_ of them&mdash;only those that will give you the level of coverage you need to assess performance and preserve good user experiences. The table below can help you determine which tools you need:
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>CrUX on BigQuery</th>
+        <th>CrUX API</th>
+        <th>PSI API</th>
+        <th>web-vitals.js</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Origin-level data (Field only)</td>
+        <td>URL and Origin-level data (Field only)</td>
+        <td>URL and Origin-level data (Lab and Field)</td>
+        <td>URL-level data (Field only)</td>
+      </tr>
+      <tr>
+        <td>Monthly aggregated data</td>
+        <td>28 days rolling</td>
+        <td>28 days rolling</td>
+        <td>Real-time</td>
+      </tr>
+      <tr>
+        <td>Allows slicing of data in meaningful ways which can be joined with other public datasets like the <a href="https://httparchive.org/">HTTP Archive</a> for advanced insights.</td>
+        <td>RESTful access to CrUX data programmatically. More filtering possibilities and faster than PSI API (also higher quota).</td>
+        <td>RESTful access to CrUX and Lighthouse data programmatically.</td>
+        <td>Collect real-time Core Web Vitals metrics from real user experiences on a website (needs to be integrated in pages).</td>
+      </tr>
+      <tr>
+        <td>Recommended for most accurate insights. Default users can query 1TB of data per month. Beyond that, the standard rates apply.</td>
```

Comment:
I'd remove "Recommended for most accurate insights.".

It's not actually more accurate that the CrUX or PSI API, so mentioned that here might make people assume something that's not true.